### PR TITLE
Added new storage bot

### PR DESCRIPTION
**Pull request by chrisbeckett**

### CODE REVIEW

File: dome9CloudBots/bots/storage_account_disable_public_network_access.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# What it does: Restricts Storage account access to internal subnets only
```
Comment (RavidAtias) [inline dome9CloudBots/bots/storage_account_disable_public_network_access.py:1]: please add this documentation to the bots.md file

File: dome9CloudBots/bots/storage_account_disable_public_network_access.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# What it does: Restricts Storage account access to internal subnets only
+# Usage: AUTO: storage_account_disable_public_network_access <vnet resource group> <vnet> <subnet>
```
Comment (RavidAtias) [inline dome9CloudBots/bots/storage_account_disable_public_network_access.py:2]: No need to use 'AUTO: ' convention in usage and example. While supporting it we prefer the usage of the remediation in our system

File: dome9CloudBots/bots/storage_account_disable_public_network_access.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        '/providers/Microsoft.Network/virtualNetworks/' + \
+        vnet_name + '/subnets/' + subnet_name
+
+    if not subscription_id or not credentials:
+        return raise_credentials_error()
+
+    try:
+        storage_client = StorageManagementClient(credentials, subscription_id)
+        storage_client.storage_accounts.update(group_name,storage_account_name, StorageAccountUpdateParameters(network_rule_set=NetworkRuleSet(default_action='Deny', virtual_network_rules=[VirtualNetworkRule(virtual_network_resource_id=subnet_path)])))
+        id = entity['id']
```
Comment (RavidAtias) [inline dome9CloudBots/bots/storage_account_disable_public_network_access.py:39]: You don't use this variable

File: dome9CloudBots/bots/storage_account_disable_public_network_access.py (hunk 1 above)
Comment (chrisbeckett) [reply dome9CloudBots/bots/storage_account_disable_public_network_access.py:1]: done

File: dome9CloudBots/bots/storage_account_disable_public_network_access.py (hunk 2 above)
Comment (chrisbeckett) [reply dome9CloudBots/bots/storage_account_disable_public_network_access.py]: Amended

File: dome9CloudBots/bots/storage_account_disable_public_network_access.py (hunk 3 above)
Comment (chrisbeckett) [reply dome9CloudBots/bots/storage_account_disable_public_network_access.py]: Removed

**Pull request merged**